asyncpg/connection.py

```diff
@@ -423,6 +423,7 @@ def cursor(self, query, *args, prefetch=None, timeout=None):
             ...         'mytable', columns=('foo', 'bar'),
             ...         output='file.csv', format='csv')
             ...     print(result)
+            ...
```

Comment:
Let's not change any lines of code/comments/docstrings that aren't directly relevant to the PR.

Reply:
I created a separated commit for it (to not mix it with lines in the commit relevant to the PR).

Does it worth a separate PR? The git history does not track PRs (may be except ranges between GPG signatures).